packages/core/src/Stores/Helpers/client-notifications.js

```diff
@@ -534,6 +535,11 @@ const checkAccountStatus = (
 
     addVerificationNotifications(identity, document, addNotificationMessage);
     const has_risk_assessment = getRiskAssessment(account_status);
+    const needs_poa = is_10k_withdrawal_limit_reached && needs_verification.includes('document');
```

Comment:
@farrah-deriv So we don't need to check the expiration and pending status of the document for poa and poi anymore?
'(document?.status !== 'expired' || document?.status !== 'pending')`?

Reply:
we have a different notification if poi/poa needs authentication or if the documents have expired

Reply:
wise ✌️